lerna.json

```diff
@@ -7,7 +7,7 @@
   "version": "independent",
   "command": {
     "version": {
-      "allowBranch": "master"
+      "allowBranch": "version-bump"
```

Comment:
Should this be `main` now?

Reply:
main is a protected branch, lerna needs the ability to push directly to the repo after creating a new version.

So the idea is to run `yarn lerna version`, which will push up to this branch, and then get another reviewer to approve the publish by merging the PR from `version-bump` to `main`.

Github Actions (or teamcity, with our internal projects), will run a publish job automatically to publish the newly bumped package versions after the merge to main happens.